src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -1435,45 +1437,39 @@ SelectionGroup = SubResource( 19 )
 [node name="bindingAgent" parent="MicrobeEditorGUI/CellEditor/LeftPanel/MainPanel/VBoxContainer/MarginContainer/EditingPanel/Structure/ScrollContainer/MarginContainer/PartsSelection/Organelle/Clip/GridContainer" index="1" groups=[
 "PlaceablePartSelectionElement",
 ] instance=ExtResource( 12 )]
-margin_right = 98.0
```

Comment:
Why did you change the organelle order in the editor? It doesn't change anything ingame, but is this an unintened change?

Reply:
Automatic change, will try to see if these are revertable.

Reply:
Tried reverting it and seems not possible, Godot is persistent with the changes the second I modified the editor tabs visibility.

Reply:
Updating the margins, happens pretty often. It seems that very often Godot doesn't realize the changes it wants to make automatically *unless* you restart Godot.